### ISSUE TITLE
Varnish 503 error on admin pages

### ISSUE THREAD

**Issue by emmajane** [humanitarianresponse/site#316]
We have had several reports of Varnish 503 errors from S2 when using admin pages (upload files; editing OG menus). I have not been able to replicate the issue when connecting to S1.

![unnamed](https://cloud.githubusercontent.com/assets/1299320/12782152/29e5d8cc-ca71-11e5-92c5-5eac67aa0bf1.gif)

![unnamed-1](https://cloud.githubusercontent.com/assets/1299320/12782162/36c4d58e-ca71-11e5-8322-e0119b6d9ad1.png)

![unnamed](https://cloud.githubusercontent.com/assets/1299320/12782155/2d554222-ca71-11e5-8043-2d4dec6bc303.png)

**Comment by pcambra** [comment]
These are due to memory problems on loading the top menu, here's a related issue https://github.com/humanitarianresponse/site/issues/241

**Comment by emmajane** [comment]
- the email re. menu editing was at 3:29PM yesterday (2 feb).
- the email re. uploading files was at 1:15AM today (3 feb)

**Comment by emmajane** [comment]
Note from Pedro re. log files:

````
[Tue Feb 02 10:01:37 2016] [error] [client 127.0.0.1] PHP Fatal error:  Allowed memory size of 536870912 bytes exhausted (tried to allocate 29016602 bytes) in /var/aegir/platforms/humanitarianresponse-7.x-2.x-prod/sites/all/modules/contrib/memcache/dmemcache.inc on line 65, referer: https://www.humanitarianresponse.info/en/group/node/33204/admin/menus
````

**Comment by emmajane** [comment]
Summary of conversation with Pedro:

This is not consistently reproduced and may be related to peak usage of the site. The work-around is to wait a period of time (traffic on the site decreases), freeing up some available memory. Then refresh the page and attempt the submission again. This is not a solution, nor is it even close to an ideal editor experience; however, it is the work-around available until #241 is addressed.
